lib/email.tsx

```diff
@@ -0,0 +1,67 @@
+import SendGrid from "@sendgrid/mail";
+import WelcomeEmail from "@/emails/welcome-email";
+import { render } from "@react-email/components";
+import { REPLY_EMAIL } from "./email/utils";
+import PendingMemberEmail from "@/emails/pending-member-email";
+SendGrid.setApiKey(process.env.SENDGRID_API_KEY);
+
+export interface SendConfirmationEmailProps {
+  email: string;
+  name: string;
+  location: string;
+  link: string;
+  recordID: string;
+  title?: string; // title is only non-required freeform field on sign up
+}
+
+export async function sendConfirmationEmails({
+  email,
+  recordID,
+  name,
+  location,
+  link,
+  title,
+}: SendConfirmationEmailProps) {
+  try {
+    // TODO better error handling, thrown in `create-member`
+    if (!recordID) throw new Error("No recordID provided");
+    if (!email) throw new Error("No email provided");
+    if (!name) throw new Error("No name provided");
+    if (!location) throw new Error("No location provided");
+    if (!link) throw new Error("No link provided");
+
+    await SendGrid.send({
+      from: {
+        email: REPLY_EMAIL,
+        name: "Hawaiians in Tech",
+      },
+      to: email,
+      subject: "Welcome to Hawaiians in Tech",
+      html: render(
+        <WelcomeEmail email={email} name={name} recordID={recordID} />,
+      ),
+    });
+
+    await SendGrid.send({
+      from: {
+        email: REPLY_EMAIL,
+        name: "Hawaiians in Tech",
+      },
+      to: email,
```

Comment:
Aiya never caught this on the first go-around, I think these are supposed to go to the admins right?
```suggestion
      to: ADMIN_EMAILS,
```

Reply:
Ha! Glad you caught this. Will fix.